system.go

```diff
@@ -28,6 +28,32 @@ type SystemInput struct {
 	Note        *string          `json:"note,omitempty"`
 }
 
+func (s *System) GetTag(tagId ID, client *Client) *Tag {
+	if s == nil {
```

Comment:
```suggestion
	if s.ID == nil {
```

Reply:
I could add a `s.ID == ""` check here.
The aim of the `if s == nil` check is to catch the unlikely event that
`var mySystem *opslevel.System` is declared and the pointer is never given a value, but later used to call GetTag()

Reply:
+1 for also a checking for `s.ID == ""`, it could lead to an annoying bug down the road.